src/equations/numerical_fluxes.jl

```diff
@@ -261,6 +261,25 @@ function Base.show(io::IO, d::DissipationLaxFriedrichsEntropyVariables)
     print(io, "DissipationLaxFriedrichsEntropyVariables(", d.max_abs_speed, ")")
 end
 
+@doc raw"""
+    DissipationMatrixWintersEtal()
+
+Creates the Roe-like entropy stable matrix dissipation operator from Winters et al. (2017). This operator
+must be used together with an entropy-conservative two-point flux function 
+(e.g., [`flux_ranocha`](@ref) or [`flux_chandrashekar`](@ref)) to yield 
+an entropy-stable surface flux. The surface flux function can be initialized as:
+```julia
+flux_es = FluxPlusDissipation(flux_ec, DissipationMatrixWintersEtal())
+```
+This implementation is adapted from the [Atum.jl library](https://github.com/mwarusz/Atum.jl/blob/c7ed44f2b7972ac726ef345da7b98b0bda60e2a3/src/balancelaws/euler.jl#L198).
```

Comment:
If we decide to include the 3D we might also want to mention that the 1D/2D came from Atum.jl and the 3D was helped by FLUXO (https://github.com/project-fluxo/fluxo/blob/master/src/equation/navierstokes/riemann.f90#L901)